bower.json

```diff
@@ -13,6 +13,7 @@
     "d3": "4.11.0",
     "mousetrap": "1.6.1",
     "normalize-css": "7.0.0",
-    "oboe": "2.1.4"
+    "oboe": "2.1.4",
+    "js-cookie": "2.2.1"
```

Comment:
@davideschiera we're adding this dependency here, even though it **is not** referenced in this repo's codebase but rather is referenced in sysdig-inspect-web [here](https://github.com/draios/sysdig-inspect-web/pull/2/files#diff-6d57078a817396eafc021282e6c8b2db).
Is this approach as designed?

Reply:
Given the current state, this is now _more_ correct. All good!